sqlx-postgres/src/advisory_lock.rs

```diff
@@ -98,8 +98,7 @@ impl PgAdvisoryLock {
     /// [hkdf]: https://datatracker.ietf.org/doc/html/rfc5869
     /// ### Example
     /// ```rust
-    /// # extern crate sqlx_core as sqlx;
-    /// use sqlx::postgres::{PgAdvisoryLock, PgAdvisoryLockKey};
+    /// use sqlx_postgres::{PgAdvisoryLock, PgAdvisoryLockKey};
```

Comment:
The example should not use `sqlx_postgres` directly. Its API isn't stable. It should be possible to just use `sqlx` itself as a dev-dependency like in `sqlx-core`: https://github.com/launchbadge/sqlx/blob/main/sqlx-core/Cargo.toml#L93

Reply:
Ok, I will revert this commit and do it using the `dev-dependency` approach then

Reply:
Done